assets/src/userTestGroups.ts

```diff
@@ -0,0 +1,12 @@
+import appData from "./appData"
+
+function getTestGroups(): string[] {
+  const groups = appData()?.userTestGroups
+  const testGroups: string[] = groups === undefined ? [] : JSON.parse(groups)
```

Comment:
Since `JSON.parse` returns an `any` type TypeScript doesn't have a problem with this, but it would still be nice to do some runtime checks to make sure the result of parsing is actually an array of strings like we'd expect.

Reply:
Typescript can't infer based on the `string[]` annotation?

Could/should this be done with superstruct in some form?

Reply:
I'm curious what you think of the revision!

Reply:
I hadn't really be familiar with `create` as opposed to `assert` and `is` previously but I think that looks good!